### ISSUE TITLE
idea: Optional logging to `tensorboard`

### ISSUE THREAD

**Issue by rparrett** [hvoss-techfak/AutoForge#14]
I've been poking around at the hyperparams a bit, suspecting that some tweaks to the tau warmup/decay could yield some small efficiency improvements. [`tensorboard`](https://www.tensorflow.org/tensorboard) seems super useful for comparing batches of experiments. (spoiler: the values you chose seem really good, lol) 

Implementation shown below is very rough, but could be cleaned up.

https://github.com/user-attachments/assets/8c483764-5846-4212-b1a6-7ec18bb85130

**Comment by hvoss-techfak** [comment]
That looks very promising. I'll try to implement the tensorboard logging feature this weekend. Do you already have some functioning code somewhere?

**Comment by rparrett** [comment]
I will clean it up a touch and share the branch.
